main.go

```diff
@@ -76,7 +78,7 @@ func main() {
 	factory := k8s.NewFunctionFactory(clientset, deployConfig)
 
 	bootstrapHandlers := bootTypes.FaaSHandlers{
-		FunctionProxy:        handlers.MakeProxy(functionNamespace, cfg.ReadTimeout),
+		FunctionProxy:        proxy.NewHandlerFunc(cfg.ReadTimeout, &handlers.FunctionLookup{}),
```

Comment:
@LucasRoesler do you mean that NewHandlerFunc only needs a `nil` parameter here?

Reply:
No, I think this line is fine, i only mean that lines 23-30 in `handlers/lookup.go` aren't needed